spotbugs/src/main/java/edu/umd/cs/findbugs/ba/npe/NullDerefAndRedundantComparisonFinder.java

```diff
@@ -325,6 +328,16 @@ public void reportBugs(Map<ValueNumber, NullValueUnconditionalDeref> nullValueGu
                         if (variableAnnotation != null) {
                             break;
                         }
+                        Instruction instruction = loc.getHandle().getInstruction();
+                        if (instruction instanceof InvokeInstruction) {
+                            InvokeInstruction invokeInstruction = (InvokeInstruction) instruction;
+                            ConstantPoolGen cpg = classContext.getConstantPoolGen();
+                            String className = invokeInstruction.getClassName(cpg);
+                            String methodName = invokeInstruction.getMethodName(cpg);
+                            String methodSig = invokeInstruction.getSignature(cpg);
+                            variableAnnotation = MethodAnnotation.fromCalledMethod(className, methodName, methodSig, false);
```

Comment:
The last argument should be `instruction instanceof INVOKESTATIC` (instead of `false`)